### PR TITLE
Fix Readme to make elixir task work with newest version

### PR DESCRIPTION
**Pull request by okaufmann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
-[![Latest Stable Version](https://poser.pugx.org/mariuzzo/laravel-js-localization/v/stable.svg)](https://packagist.org/packages/mariuzzo/laravel-js-localization)
-[![Total Downloads](https://poser.pugx.org/mariuzzo/laravel-js-localization/downloads.svg)](https://packagist.org/packages/mariuzzo/laravel-js-localization)
-[![License](https://poser.pugx.org/mariuzzo/laravel-js-localization/license.svg)](https://packagist.org/packages/mariuzzo/laravel-js-localization)
+![Laravel 5.1](<https://img.shields.io/badge/Laravel-5.1-f4645f.svg>)
 
-This is a simple package that convert all your localization messages of your Laravel app to JavaScript, and provides a small JavaScript library to interact with those messages.
+![Laravel 5.0](<https://img.shields.io/badge/Laravel-5.0-f4645f.svg>)
+
+![Laravel 4.2](<https://img.shields.io/badge/Laravel-4.2-f4645f.svg>)
+
```
Comment (rmariuzzo) [inline README.md]: @okaufmann separating the badges by empty lines will make each badges one above the other. I think it is better to not separate and keep them visually in the same line. What do you think?

File: README.md [hunk 2]
```diff
@@ -18,106 +23,143 @@ Installation
 ------------
 
 Add the following line to you `composer.json` file under `require`.
-```json
```
Comment (rmariuzzo) [inline README.md]: @okaufmann it looks better to use the code hints, it helps GH to add syntax highlighting to the source codes. What do you think?

File: README.md [hunk 3]
```diff
@@ -18,106 +23,143 @@ Installation
...
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-});
-```
 Documentation
 -------------
 
-This is the documentation regarding the thin JavaScript library. The library highly inspired on Laravel's `Lang` class.
+This is the documentation regarding the thin JavaScript library. The library
+highly inspired on Laravel's `Lang` class.
```
Comment (rmariuzzo) [inline README.md]: @okaufmann: are you using an editor that wrap all the texts?

File: README.md (hunk 1 above)
Comment (okaufmann) [inline README.md]: I like it in a row too. I did not want to change the badges. You can revert this sorry.

File: README.md (hunk 2 above)
Comment (okaufmann) [inline README.md]: As you mentioned on the last pull request I used to use the code hints. I dod not touch this line why that is gone ? 0.o

File: README.md (hunk 3 above)
Comment (okaufmann) [inline README.md]: Ah yes i use [texts](http://www.texts.io/). is this a problem ?

**Pull request closed without merging**